src/zc/sourcefactory/policies.py

```diff
@@ -54,7 +55,8 @@ def getTitle(self, value):
         if md:
             title = md.title
         else:
-            title = value.decode() if isinstance(value, bytes) else unicode(value)
+            title = value.decode() if isinstance(value,
+                                                 bytes) else unicode(value)
```

Comment:
Interesting way to format. Maybe it would be more clear with a proper if/else structure.

Reply:
Fixed in https://github.com/zopefoundation/zc.sourcefactory/pull/7/commits/9928a0a306dc11092171aa039c18b8048f3d63d9